src/main/kotlin/io/libp2p/pubsub/gossip/GossipScore.kt

```diff
@@ -193,6 +197,10 @@ class GossipScore(
         }
     }
 
+    fun getCachedScore(peerId: PeerId): Double {
```

Comment:
I'd better return `Optional<Double>` here since `0.0` score is not the same as absence of score

Reply:
Isn't 0 a reasonable default?  This should only show up in the brief window between the time a peer connects and we requests its score.

Reply:
Agree. I was actually thinking about the case when score requested for unknown peer. But `0.0` default indeed looks fine as if/when a peer is connected it gets `0.0` score by default. Se we may assume any unknown peer has score `0.0`